gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/HadoopFileSystemTestBase.java

```diff
@@ -364,6 +369,112 @@ public void testOverwrite()
     // Try to create the same file again with overwrite == false.
     assertThrows(
         IOException.class, () -> ghfsHelper.writeFile(hadoopPath, text, 1, /* overwrite= */ false));
+
+    // Try to create the same file again with overwrite == true.
+    ghfsHelper.writeFile(hadoopPath, text, 1, /* overwrite= */ true);
```

Comment:
You need to write some different `text` here to test that files actually overridden with a new content.

Reply:
Done.